adm/unix/compileunix.py

```diff
@@ -42,14 +42,14 @@ def gcc(exeFile, srcFiles, includes, libs, defs):
 
   cmd += " -DPLAT_BUILD_VERSION=" + '\\"' + env.buildVersion() + '\\"'
 
-  # libs
-  for lib in libs:
-    cmd += " -l" + lib
-
   # src
   for src in srcFiles:
     cmd += " " + src
 
+  # libs
```

Comment:
I guess I made this change before and has pushed to dev branch. 

@divisuals  and I decide to follow some [git branching practice](https://nvie.com/posts/a-successful-git-branching-model/?). next time, when you want to work on something, please make a branch based on dev branch, then after the pull request is completed, we will merge it into dev branch first, and before a public release, we will merge dev branch to master branch.

Reply:
Maybe it'd be better for me just to rebase my changes on top of develop branch and make another pull-request.